.github/workflows/tests.yml

```diff
@@ -2,6 +2,10 @@ name: Tests
 
 on:
   pull_request:
+  push:
```

Comment:
IMO we don't need this. Workflow is already started on pull requests, so for manually committed changes it should be enough to run all tests once per day.

Reply:
I think there's definitely value in running the test suite after merging a PR into into master or a release branch? It's a bit odd not to. 

We can add a specific filter to only run it on commits to master or release branches if that's better?

Reply:
If we want to move all builds to GitHub actions in the future, we need to always specify a branch. Because matrices are different. In this PR I would like to move only [javascript-master](https://djangoci.com/job/javascript-master/) so both `push` and `schedule` should be run only on master.

Reply:
Ok! Makes sense. I've added that 👍